CHANGELOG.md

```diff
@@ -117,7 +117,6 @@
 - use of maximumSelectionSize from plone in ObjectBrowserWidget. @giuliaghisini
 - Added selectableTypes in ObjectBrowserWidget @giuliaghisini
 
-
```

Comment:
@iFlameing same here.

Reply:
Prettier not allowing me to override this. It is not accepting the extra space

Reply:
Fair enough. I won't argue with Prettier. :)